### PR TITLE
new post about monica's talk on march 14

### PR DESCRIPTION
**Pull request by mokaixu**
Post for the tech talk as requested :)

### CODE REVIEW

File: content/W2017/W17-event-beckwith-talk.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+Title: Tech Talk: Hotspot Virtual Machine's Execution Engine by Monica Beckwith
```
Comment (evykassirer) [inline content/W2017/W17-event-beckwith-talk.md:1]: I like the idea of adding out of town, like in http://wics.uwaterloo.ca/2016/02/stephanie-morillo-talk/

`Out of Town Speaker: Hotspot Virtual Machine's Execution Engine by Monica Beckwith`

File: content/W2017/W17-event-beckwith-talk.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+Date: 2017-03-14
+Category: Events
+Tags: events, academic
```
Comment (evykassirer) [inline content/W2017/W17-event-beckwith-talk.md:4]: 'talks'

File: content/W2017/W17-event-beckwith-talk.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+Slug: beckwith-tech-talk-w17
+Author: Monica Xu
+Summary: Come listen about the amazing work Monica Beckwith is doing!
+
+Monica Beckwith has worked with the Java Virtual Machine
```
Comment (evykassirer) [inline content/W2017/W17-event-beckwith-talk.md:9]: can you add an intro before this paragraph that looks like the first paragraph in http://wics.uwaterloo.ca/2016/02/stephanie-morillo-talk/ ?

File: content/W2017/W17-event-beckwith-talk.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+Title: Tech Talk: Hotspot Virtual Machine's Execution Engine by Monica Beckwith
+Date: 2017-03-14
+Category: Events
+Tags: events, academic
+Slug: beckwith-tech-talk-w17
+Author: Monica Xu
+Summary: Come listen about the amazing work Monica Beckwith is doing!
```
Comment (evykassirer) [inline content/W2017/W17-event-beckwith-talk.md:7]: come listen **to**

File: content/W2017/W17-event-beckwith-talk.md (hunk 3 above)
Comment (evykassirer) [reply content/W2017/W17-event-beckwith-talk.md:9]: and then "Abstract:" before this line

**Review by evykassirer — CHANGES_REQUESTED** [review]
thanks for putting this up! it was on my todo list for this morning but I'm grateful you took it on ❤️ 

overall looks great - left a few comments for changes

File: content/W2017/W17-event-beckwith-talk.md (hunk 2 above)
Comment (annalorimer) [reply content/W2017/W17-event-beckwith-talk.md:4]: Adding on to Evy's comment: 

Replace 'events' with 'talks'.

File: content/W2017/W17-event-beckwith-talk.md (hunk 4 above)
Comment (annalorimer) [reply content/W2017/W17-event-beckwith-talk.md:7]: The summary is basically like a headline, so we want to include more information about the talk. Have a look at the Leigh Honeywell post, there's a good example there.

File: content/W2017/W17-event-beckwith-talk.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+but also improving the Just-in-time (JIT) code quality for 
+various processor architectures as well as working with the 
+garbage collectors and improving garbage collection for server systems.
+
+During this talk, Monica will cover a few JIT and Runtime 
+optimizations and she will dive into the HotSpot garbage 
+collection and provide an overview of the various garbage
+collectors available in HotSpot.
+
+Note: Please claim your free admission ticket [here]
```
Comment (annalorimer) [inline content/W2017/W17-event-beckwith-talk.md:20]: 'Please claim your free admission ticket [here]' -> 'Registration is required. Tickets are available [here]"

File: content/W2017/W17-event-beckwith-talk.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+(https://www.eventbrite.com/e/monica-beckwith-hotspot-
+virtual-machines-execution-engine-talk-tickets-32482143967)
+
+## Event Details ##
+
++ **Who:** Everyone
++ **What:** Tech Talk
```
Comment (annalorimer) [inline content/W2017/W17-event-beckwith-talk.md:27]: Again, have a look at the Leigh Honeywell post for a good idea of how to use this section 😄

File: content/W2017/W17-event-beckwith-talk.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
++ **Where:** Science Teaching Complex 0040
```
Comment (annalorimer) [inline content/W2017/W17-event-beckwith-talk.md:28]: Add STC in brackets after the building name.

**Review by annalorimer — CHANGES_REQUESTED** [review]

**Comment by annalorimer** [comment]
Actually, @fat1996 already made an event post. I reverted master because we wanted other events to be at the top of the page.

**Comment by annalorimer** [comment]
Closing this issue since we already have an event post. 

Sorry @mokaixu you went and did all this work 😞.

**Comment by mokaixu** [comment]
Hi Just saw the requests! No problem @annalorimer It was good practise :)

LOL very efficient now after all that troubleshooting :')

**Pull request closed without merging**